src/main/java/com/google/api/codegen/gapic/MainGapicProviderFactory.java

```diff
@@ -421,6 +427,16 @@
           providers.add(messageProvider);
         }
       }
+      if (generatorConfig.enableTestGenerator() && id.equals(RUBY)) {
+        GapicProvider<?> testsProvider =
+            ViewModelGapicProvider.newBuilder()
+                .setModel(model)
+                .setApiConfig(apiConfig)
+                .setSnippetSetRunner(new CommonSnippetSetRunner(snippetContext))
+                .setModelToViewTransformer(new RubyTestsTransformer())
```

Comment:
Consistency is good, but what's the `GapicSurface` part in those names for? After renaming `RubyTestsTransformer` should I leave a TODO to remove that part from the names?